src/resolvers/nodeResolver.ts

```diff
@@ -27,7 +27,7 @@ const versionCache: {
 			.get(1)
 			.then((res: { results: Array<{ name: string }>; next?: string }) => {
 				const curr = _(res.results).map('name').filter(versionTest).value();
-				const tags = prev.concat(curr);
+				const tags = prev.concat(curr as string[]);
```

Comment:
Can you use
```typescript
const curr: string[] = _(res.results).map('name').filter(versionTest).value();
```
instead so that `curr` is typed correctly everywhere rather than being specifically cast just here